tests/python/bandit_test_files/exec-py2.py

```diff
@@ -0,0 +1,2 @@
+exec("do evil")
+exec "do evil"
```

Comment:
it is tested, under python3 which our test cases run the file fails to parse, bandit has an extra section for such kind of errors which I parse, so this file is totally on purpose^^